### ISSUE TITLE
Inference from trained model

### ISSUE THREAD

**Issue by jessalva** [adrianodennanni/pre-trained-nn-benchmark#1]
Please help me on how to draw inference from the trained model for image captcha.

**Comment by adrianodennanni** [comment]
Please, could you elaborate your question with more information?

**Comment by jessalva** [comment]
How can I get the predicted label for a particular image.

**Comment by adrianodennanni** [comment]
You should use the `predict` method for Keras models: https://keras.io/models/model/#predict

**Comment by jessalva** [comment]
Can you send the code for the predicted label of a particular image?

Like after converting the image to a tf record,
Can you display the image, the predicted captcha and actual captcha?

**Comment by adrianodennanni** [comment]
This repo is an experiment for benchmarking pre-trained neural networks. I never used it in production, so I never wrote the code you are asking for.

I recommend you to search tutorials for what you need in google: https://www.google.com/search?q=how+to+load+a+image+in+keras
